src/content/app/species-selector/views/species-selector-results-view/SpeciesSelectorResultsView.tsx

```diff
@@ -73,19 +70,21 @@ const Content = (props: { onClose: () => void }) => {
     for (const addedGenome of genomes) {
       trackAddedGenome(addedGenome);
     }
+
+    navigate('/species-selector');
```

Comment:
I am very unsure and confused about how the navigation should work when user presses the "Add" button. Last I heard, Andrea was saying that the user should be taken to the screen they came from (which, if they pressed on the "Assemblies" link on the species page, will not be "/species-selector"). I do not know how much of this is still true. You can discuss this with Andrea.